packages/cli/src/api/test-run.api.ts

```diff
@@ -91,7 +91,12 @@ export const getCachedTestRunResults = async ({
       ?.results ?? [];
 
   // Only return passing tests
-  return results.filter(({ result }) => result === "pass");
+  return (
+    results
+      .filter(({ result }) => result === "pass")
+      // If it passed it can't have any screenshot diffs:
+      .map((result) => ({ ...result, screenshotDiffResults: [] }))
```

Comment:
Note: I think `ScreenshotDiffResultMissingBase` would still pass.

Reply:
Ah that's true :/. Hmm I could do a BE call to load the information.